rc/lsp.kak

```diff
@@ -1127,6 +1127,42 @@ method    = "rust-analyzer/inlayHints"
 ' "${kak_session}" "${kak_buffile}" "${kak_opt_filetype}" "${kak_timestamp}" | eval "${kak_opt_lsp_cmd} --request") > /dev/null 2>&1 < /dev/null & }
 }
 
+# texlab extensions
+
+define-command texlab-forward-search -docstring "texlab-forward-search: Execute SyncTeX Forward Search for current line (texlab)" %{
```

Comment:
I believe we have an unwritten convention that we use a `-docstring "texlab-forward-search:` synopsis only if there are any parameters. Wil change.